content/en/blog/2022/jaeger-native-otlp/index.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Supporting Jaeger with native OTLP
+linkTitle:
```

Comment:
have something short here, like "Jaeger OTLP support"

Reply:
Oops, I think I had intended to come back to that and overlooked it. Thanks!